### DISCUSSION TITLE
Welcome to ultimate-react-hook-form-form Discussions!

### DISCUSSION THREAD

**Discussion by satansdeer** [satansdeer/ultimate-react-hook-form-form#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by antoniobfm** [comment]
First of all, thank you for the video! :)

I used the latest version of all dependencies, so there was some things i needed to change to get everything working.

Do you think it would be useful to create a pull request with all these changes?
